translations/de/info/task_description.html

```diff
@@ -24,101 +22,100 @@
-            little doll, and was now on the table, merrily running round and round
-            after her own shawl, which was trailing behind her.
+            ... »Und was Euch betrifft,« fuhr Alice fort und wandte sich wütend an die Schwarze
+            Königin, die sie für die Ursache des ganzen Unfugs hielt – aber die Königin war
+            nicht mehr an ihrer Seite – sie war plötzlich auf die Größe einer kleinen Puppe
+            geschrumpft und befand sich jetzt auf dem Tisch, wo sie fröhlich im Kreis ihrem
+            Schal nachlief, der hinter ihr herschleifte.
         </p>
 
         <p>
-            At any other time, Alice would have felt surprised at this, but she was
-            far too much excited to be surprised at anything NOW. 'As for YOU,'
-            she repeated, catching hold of the little creature in the very act of
-            jumping over a bottle which had just lighted upon the table, 'I'll shake
-            you into a kitten, that I will!'
+            Zu jeder anderen Zeit wäre Alice darüber verwundert gewesen, aber sie war viel
+            zu aufgeregt, um sich jetzt über irgendetwas zu wundern. »Was dich betrifft,«
+            wiederholte sie und kriegte das kleine Geschöpf zu pakken, als es gerade über
+            eine Flasche sprang, die eben auf dem Tisch gelandet war, »ich schüttele dich,
+            bis du ein Kätzchen bist!«
         </p>
+
         <img src="{{ MEDIA }}alice-kitty-sepia.png" class="shadow right" width="220px">
 
         <p class="title">
-            CHAPTER X. Shaking
+            Kapitel X. Schütteln
         </p>
 
         <p>
-
-
-            She took her off the table as she spoke, and shook her backwards and
-            forwards with all her might.
-
-            The Red Queen made no resistance whatever; only her face grew very
-            small, and her eyes got large and green: and still, as Alice went on
-            shaking her, she kept on growing shorter--and fatter--and softer--and
-            rounder--and--
-
+            Sie nahm die Königin vom Tisch, während sie sprach, und schüttelte sie mit aller
+            Kraft vor und zurück.<br>
+            Die Schwarze Königin leistete überhaupt keinen Widerstand; nur ihr Gesicht wurde
+            sehr klein und ihre Augen wurden groß und grün; und als Alice sie weiter schüttelte,
+            fuhr sie fort, kleiner zu werden – und fetter – und weicher – und runder – und –
         </p>
 
         <p class="title">
-            CHAPTER XI. Waking
+            Kapitel XI. Erwachen
         </p>
 
         <p>
-            --and it really WAS a kitten, after all.
+            – und es war zu guter Letzt wirklich ein Kätzchen
         </p>
         <p style="float: right">
-            "Through the Looking-Glass." Lewis Carroll
+            "Alice hinter den Spiegeln" von Lewis Carroll<br>
+            Aus dem Englischen von Jörg Karau, 2008
```

Comment:
that's great...\*\*starts a google-search\*\*  congrats for discovering joergkarau-texte.de! 👾